app/src/main/java/com/example/scoutconcordia/DES.java

```diff
@@ -124,19 +124,21 @@ private static String bytesToHex(byte[] bytes) {
         return data;
     }
 
+
```

Comment:
within DES the encryptFile and lockFile appear to have the same functionality with the only difference being the output however I believe it is safe to assume for any encryption / decryption the file should rename named the same. The same applies for decryptFile and unlockFile.